packages/remix-forms/src/schema-form.tsx

```diff
@@ -338,13 +343,14 @@ function SchemaForm<Schema extends FormSchema>({
       const deepHiddenFieldsErrors = hiddenFields?.map((hiddenField) => {
         const hiddenFieldErrors = fieldErrors(hiddenField)
 
-        if (hiddenFieldErrors instanceof Array) {
+        if (Array.isArray(hiddenFieldErrors)) {
```

Comment:
Seems pretty safe:
https://stackoverflow.com/questions/22289727/difference-between-using-array-isarray-and-instanceof-array